omnibus/files/private-chef-cookbooks/private-chef/templates/default/oc_erchef.config.erb

```diff
@@ -197,7 +197,7 @@
            {ip_mode, [ <%=PrivateChef['use_ipv6'] ? "ipv6,ipv4" : "ipv4" %> ] },
            %% Database connection parameters
            {db_host, "<%= node['private_chef']['postgresql']['vip'] %>"},
-           {db_port, 5432},
+           {db_port, <%= node['private_chef']['opscode-erchef']['sql_user'] %>},
```

Comment:
D'oh. Corrected.   As we move forward with different permitted  postgres backends per-service, we'll actually push the postgres host/vip/superuser  config down into the application itself, probably with a default value that is used if it's not specified continuing to live in `postgresql`